notifications.bs

```diff
@@ -483,7 +516,8 @@ must be run.
 <ol>
   <li><p>Wait for any <a lt="fetch">fetches</a> to complete and
   <var>notification</var>'s <a>icon resource</a> and <a>sound resource</a>
-  to be set (if any).
+  to be set (if any), as well as the <a>action icon resource</a>s for the
```

Comment:
It's customary (at least in this spec) for plurals to be linked completely, using the `lt` attribute for the singular form.

> \<a lt="action icon resource">action icon resources\</a>

(dito on line 539)